core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -129,7 +129,10 @@ class KafkaServer(val config: KafkaConfig, time: Time = Time.SYSTEM, threadNameP
 
   private var shutdownLatch = new CountDownLatch(1)
 
+  //properties for MetricsContext
```

Comment:
jmxPrefix is from existing codebase: https://github.com/apache/kafka/blob/ec205171e5fddbae2ebc1784adf73a6732889a49/core/src/main/scala/kafka/server/KafkaServer.scala#L132

We can move the comments beneath it so it is not confusing

Reply:
Moved these definition to KafkaConfig

Reply:
any reason you moved some but not jmxPrefix? I think it would make sense to keep all the metrics context constants in one place. I didn't see any comment from @mumrah suggesting to move them.

Reply:
From his comments: "I think the ability to include "metrics.context.*" values as config is generally useful. Is it worth doing this for the broker as well?"
Looks like I misunderstood what he meant. Just talked to him on slack.

Reply:
This comment was just about the `jmxPrefix` variable. I realize it's pre-existing code, but now that it's not specific to JMX, I suggested renaming it.

Reply:
@xiaodongdu what @mumrah was saying is that we could add the properties to the kafka broker configuration and pass those values into the context, similar to what we do for client. We can update the KIP and add that.

Reply:
@xvrl  Do you mean add this logic back to KafkaServer.scala:
metadata.putAll(config.originalsWithPrefix(CommonClientConfigs.METRICS_CONTEXT_PREFIX, false));

Reply:
yes, I believe that's what @mumrah meant. If we decide to do this, I'll update the KIP and send a follow-up email with some of the other tweaks we discussed in this PR.

Reply:
renamed jmxPrefix to metricsPrefix and add broker MetricsContext properties